examples/simple-asset-transfer/quorum/api/utils/web3.js

```diff
@@ -1,9 +1,8 @@
 const Web3 = require('web3');
 const config = require('../config/config');
 
-const option = `ws://${config.web3.host[`${process.platform}`]}:${config.web3.port}`;
-const web3 = new Web3(option);
-
+const rpcApiHost = `http://${config.web3.host[`${process.platform}`]}:${config.web3.rpcPort}`;
```

Comment:
need to add this `rpcPort` to config. Or why not just leave it as `port`?

Reply:
> need to add this `rpcPort` to config. Or why not just leave it as `port`?

I wanted to keep it separate from the WS API / port. Naming is a little confusing because in the end both HTTP and the WS are using the same JSON RPC API under the hood just with different transport protocols (HTTP, WS).

Reply:
ok, then you need to add this `rpcPort` to `config/config`, no?